bitnami/etcd/values.yaml

```diff
@@ -286,10 +286,12 @@ lifecycleHooks: {}
 ## etcd container ports to open
 ## @param containerPorts.client Client port to expose at container level
 ## @param containerPorts.peer Peer port to expose at container level
+## @param containerPorts.metrics Metrics port to expose at container level when metrics are enabled
```

Comment:
We should note this port will only be used if `metrics.useSeparateEndpoint=true`